engine/core/source/org/pentaho/reporting/engine/classic/core/modules/output/table/xls/helper/HSSFCellStyleProducer.java

```diff
@@ -590,9 +590,28 @@ public CellStyle createCellStyle( final InstanceID id, final StyleSheet element,
       }
     }
 
-    final CellStyle hssfCellStyle = workbook.createCellStyle();
-    TextRotation rotation = null;
-    if ( element != null ) {
+    ExcelCellStyleBuilder builder = new ExcelCellStyleBuilder( this.workbook );
+
+    builder.withRotation( element );
+    builder.withElementStyle( element, styleKey );
+    builder.withBackgroundStyle( bg, styleKey );
+
+    final CellStyle hssfCellStyle = builder.build();
+
+    /*final CellStyle hssfCellStyle = workbook.createCellStyle();
```

Comment:
Dont comment out code, delete it. If we need to look at the old code, we can look at Git's history.